test/14_release-pinning.ts

```diff
@@ -39,6 +40,24 @@ describe(`Tracking latest release`, () => {
 			passthrough: { user: admin },
 		});
 
+		const org = await fx.organizations.admin;
+		const deviceType: DeviceType = (
+			await pineUser.get({
+				resource: 'device_type',
+				options: { $select: 'id', $filter: { slug: 'intel-nuc' } },
+			})
+		).body[0];
+		applicationToDelete = await fixtures.createResource({
+			resource: 'application',
+			method: 'POST',
+			body: {
+				app_name: 'application_to_delete',
+				organization: org.id,
+				is_for__device_type: deviceType.id,
+			},
+			user: { token: admin.token! },
+		});
```

Comment:
Isn't this v working?
```suggestion
		({ body: applicationToDelete } = await pineUser.post({
			resource: 'application',
			body: {
				app_name: 'application_to_delete',
				organization: org.id,
				is_for__device_type: deviceType.id,
			}
		}).expect(200));
```

Reply:
Not exactly typing-wise. For this to work, I need to destructure into a const and then assign the value to `applicationToDelete`